elastio-s3-changelog/README.md

```diff
@@ -0,0 +1,45 @@
+# S3 Changelog
+
+This CloudFormation template enables the Changelog feature for S3 buckets.
+It deploys an SQS queue and an EventBridge rule which sends S3 update events to the queue.
+Then, the Elastio `iscan` job reads those events to perform the scanning of new objects.
+
+There are a couple of scenarios where this feature can be useful:
+
+* Incremental S3 scanning. For large buckets, scanning all objects might take a lot of time.
+    The changelog queue allows the scanning of only new objects in the bucket, significantly improving the
+    scan performance after the initial scan of the entire bucket is done.
+* Quasi-realtime scanning (QRTS). Instead of running scan jobs according to a schedule in the Protection
```

Comment:
This isn't going to be supported initially.  I don't want to confuse customers by mentioning it until it's shipped.

Reply:
Let's comment out these then